services/ProductRegistrationService.php

```diff
@@ -88,6 +88,8 @@ private function optInStrategy($email) {
         curl_setopt($curl, CURLOPT_POST, true);
         curl_setopt($curl, CURLOPT_POSTFIELDS, http_build_query(array('email' => $email)));
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYHOST, 2);
```

Comment:
Oh sorry, I just went ahead and set host to default for documentation. How can one even anticipate such behavior. The link explains a lot, I was lucky with portal as I just ignored ssl completely. Anyway thanks for clearing up.